crates/contracts/src/alloy.rs

```diff
@@ -70,3 +97,45 @@ macro_rules! bindings {
         }
     };
 }
+
+crate::bindings!(GnosisSafe);
+crate::bindings!(GnosisSafeCompatibilityFallbackHandler);
+crate::bindings!(GnosisSafeProxy);
+crate::bindings!(GnosisSafeProxyFactory);
+
+pub mod macros {
+    #[macro_export]
+    macro_rules! tx_value {
+        ($call:expr_2021, $value:expr_2021) => {{
+            const NAME: &str = stringify!($call);
+            $call
+                .value($value)
+                .send()
+                .await
+                .expect(&format!("failed to send: {}", NAME))
```

Comment:
This gonna print a pretty tx with all the fields, right?
```suggestion
                .expect(&format!("failed to send a tx: {}", NAME))
```

Reply:
`NAME` here becomes the expression call, so it will rather print the code call, example — https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=e654ccc595c48d58a6c939ff7a8f5689

Reply:
Gotcha, thanks